mta-sts.txt

```diff
@@ -251,29 +251,29 @@ sts-ext-value   = 1*(%x21-3A / %x3C / %x3E-7E)       ; chars excluding
 
 3.2.  MTA-STS Policies
 
-   The policy itself is a set of key/value pairs served via the HTTPS
-   GET method from the fixed [RFC5785] "well-known" path of ".well-
-   known/mta-sts.txt" served by the "mta-sts" host at the Policy Domain;
-   the [RFC2616] "Content-Type" header MUST be "text/plain".  Thus for
-   "example.com" the path is "https://mta-sts.example.com/.well-known/
-   mta-sts.txt".
+   The policy itself is a set of key/value pairs (similar to [RFC2822]
```

Comment:
RFC2822 is obsoleted by [RFC5322](https://tools.ietf.org/html/rfc5322) - sorry hadn't checked that before.